frontend/src/context/DatabaseProvider.jsx

```diff
@@ -17,10 +19,16 @@ export default (props) => {
   const [totalTransactions, dispatchTotalTransactions] = useState(0);
   const [totalAccounts, dispatchTotalAccounts] = useState(0);
   const [lastDayTxCount, dispatchLastDayTxCount] = useState(0);
+  const [phase2TotalStake, dispatchPhase2TotalStake] = useState(
+    "11416584959186517242479124953463"
+  );
+  const [phase2TotalVotes, dispatchPhase2TotalVotes] = useState(
+    "1272506343149920375853886366252"
```

Comment:
What are these magic numbers?

Reply:
it is from testnet vote contract, I use them as testing and later I think it is okay to be initial state.

Reply:
I would prefer either null or zero values as initial

Reply:
okay

Reply:
after re thinking, it could not be null or zero from start